tokio-console/src/conn.rs

```diff
@@ -78,7 +82,23 @@ impl Connection {
                 tokio::time::sleep(backoff).await;
             }
             let try_connect = async {
-                let mut client = InstrumentClient::connect(self.target.clone()).await?;
+                let channel = match self.target.scheme_str() {
+                    None | Some("file") => {
+                        // Dummy endpoint is ignored by the connector.
+                        let endpoint = Endpoint::from_static("http://localhost");
+                        let path = self.target.path().to_owned();
+                        endpoint
+                            .connect_with_connector(tower::service_fn(move |_| {
+                                UnixStream::connect(path.clone())
+                            }))
+                            .await?
+                    }
```

Comment:
this match arm will need to be `#[cfg(unix)]`, because it uses the Unix-only `UnixStream` type:
```suggestion
                    #[cfg(unix)]
                    None | Some("file") => {
                        // Dummy endpoint is ignored by the connector.
                        let endpoint = Endpoint::from_static("http://localhost");
                        let path = self.target.path().to_owned();
                        endpoint
                            .connect_with_connector(tower::service_fn(move |_| {
                                UnixStream::connect(path.clone())
                            }))
                            .await?
                    }
```

also, we may want to add a separate `#[cfg(not(unix))]` match arm that prints an error message when trying to connect to a Unix domain socket on non-unix platforms?

Reply:
Done.